src/pure/target.mli

```diff
@@ -101,12 +100,12 @@ module Condition : sig
   (**
     An execution anti-condition; the condition defines when a target is
     ready and therefore should be run if the condition is {emph not} met: {ul
-    {li with [`Never] the target always runs (because never “ready”),}
-    {li with [`Satisfied] the target never runs (a bit useless),}
+    {li with [`Never] the target always runs and then fails
```

Comment:
It "unlocks" it. I makes the DB consistent again and makes the user's workflow fail.